browser/greaselion/greaselion_browsertest.cc

```diff
@@ -326,3 +358,78 @@ IN_PROC_BROWSER_TEST_F(GreaselionServiceTest, IsNotGreaselionExtension) {
 
   EXPECT_FALSE(greaselion_service->IsGreaselionExtension("INVALID"));
 }
+
+#if defined(OS_WIN)
```

Comment:
why are these windows specific?

Reply:
It seemed that only Windows respected the `--lang` command-line option, which is the method I was using to change locale in the tests. Not sure if there's some better way to achieve this...

Reply:
`g_browser_process->SetApplicationLocale`

Reply:
That didn't seem to have any effect on `navigator.language` from what I was seeing.